test/e2e/agent/plugins/metrics/src/test/java/org/apache/shardingsphere/test/e2e/agent/metrics/MetricsPluginE2EIT.java

```diff
@@ -38,29 +38,29 @@
 @Slf4j
 public final class MetricsPluginE2EIT extends BasePluginE2EIT {
     
-    public static final String ROUTED_INSERT_SQL = "routed_insert_sql_total";
+    public static final String ROUTED_INSERT_SQL = "route_sql_insert_total";
     
-    public static final String ROUTED_UPDATE_SQL = "routed_update_sql_total";
+    public static final String ROUTED_UPDATE_SQL = "route_sql_update_total";
     
-    public static final String ROUTED_DELETE_SQL = "routed_delete_sql_total";
+    public static final String ROUTED_DELETE_SQL = "route_sql_delete_total";
     
-    public static final String ROUTED_SELECT_SQL = "routed_select_sql_total";
+    public static final String ROUTED_SELECT_SQL = "route_sql_select_total";
```

Comment:
These changes were made recently. https://github.com/apache/shardingsphere/pull/23553/files#diff-5a3c440d324b8e735f8bdcdda07cebdb54711c6d3130d47024646b4a753b57dc

Reply:
I see, thanks for pointing it out